src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -1053,8 +1046,7 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
                     (((uint32_t) adv_params.itvl_min) * 10) / 16, (connectable) ? "" : "non-", mDeviceName);
 
     {
-        ret = ble_gap_adv_start(own_addr_type, NULL, BLE_HS_FOREVER, &adv_params, ble_svr_gap_event, NULL);
-        if (ret == BLE_HS_EALREADY)
+        if (ble_gap_adv_active())
         {
             /* This error code indicates that the advertising is already active. Stop and restart with the new parameters */
```

Comment:
nit: As far as i see the `ble_gap_adv_active` returns `0` or `1`, so you might want to consider changing this comment about `error code`.

Reply:
Done!